libs/intl/src/Intl.tsx

```diff
@@ -40,17 +40,21 @@ interface IntlProps {
 }
 
 export default function Intl({ name, params, children }: IntlProps) {
-  const { lang, dictionary, intlUtil } = useIntlContext();
-
-  if (!dictionary?.translations?.[lang]?.[name] && !dictionary?.messages?.[name]) {
-    return <>{name}</>;
+  const intlContext = useIntlContext();
+
+  if (intlContext) {
+    const { lang, dictionary, intlUtil } = intlContext;
+    if (!dictionary?.translations?.[lang]?.[name] && !dictionary?.messages?.[name]) {
+      return <>{name}</>;
+    }
+
+    return (
+      <>
+        {intlUtil.translate({ name, params, render: children }).map((it, i) => (
+          <React.Fragment key={i}>{it}</React.Fragment>
+        ))}
+      </>
+    );
   }
-
-  return (
-    <>
-      {intlUtil.translate({ name, params, render: children }).map((it, i) => (
-        <React.Fragment key={i}>{it}</React.Fragment>
-      ))}
-    </>
-  );
+  return <></>;
```

Comment:
I would let this error out instead of returning empty tag. It doesn't make sense to use Intl tag without Intl